src/Popover.tsx

```diff
@@ -21,6 +21,7 @@ export type PopoverProps = {
   forceInitialAnimation?: boolean;
   numberOfLines?: number;
   visible?: boolean;
+  caretStyle: ViewProps['style'];
```

Comment:
`caretStyle` should be optional :)
```suggestion
  caretStyle?: ViewProps['style'];
```